recipes/confu_json/all/conanfile.py

```diff
@@ -28,39 +25,46 @@ def _minimum_cpp_standard(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "15",
+            "Visual Studio": "17",
```

Comment:
Why does it change now?

Reply:
I only recently tried to compile this library with visual c++. For me it only compiles with visual studio 17